### PR TITLE
cleanup generated oj.so in build_test script, add to .gitignore

### PR DESCRIPTION
**Pull request by fxfilmxf**
Since I made a mistake and committed oj.so in my last pull request, it seems like it should be added to .gitignore as well as cleaned up after the build_test script runs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ Makefile
 doc
 .yardoc
 Gemfile.lock
+ext/oj/oj.so
```
Comment (ohler55) [inline .gitignore]: *.so would be better

File: build_test.sh [hunk 2]
```diff
@@ -39,4 +39,6 @@ cd ../../test
 ./test_all.sh
 cd ..
 
+echo "\nRemoving test generated ext/oj/oj.so"
+rm ./ext/oj/oj.so
```
Comment (ohler55) [inline build_test.sh]: Removing after running the tests means you can not run the test twice. This does not work for me.

**Comment by fxfilmxf** [comment]
Updated after feedback from @ohler55.

**Comment by ohler55** [comment]
Thanks

**Pull request merged**